### PR TITLE
Sh to job feature

### PR DESCRIPTION
**Pull request by NoaBablikiCertora**
create jobs from shell scripts:
- initially, all the scripts in the certora/script directory will become jobs
- while working, every shell script file that is copyed into the script directory will become a job

### CODE REVIEW

**Comment by NoaBablikiCertora** [comment]
1. choose a shell script file from the browser
2. if the script is a legal certora cli shell script, it will become a new job. otherwise, there will be a popup describing the problem with the script

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file.
 
+### 0.1.3 ()
+
+## Features
+
+* Create a job from a shell script file ([#88](https://github.com/Certora/vscode-certora-prover/pull/88))
+* Added option to choose branch in production ([#87](https://github.com/Certora/vscode-certora-prover/pull/87))
```
Comment (sarittenen) [inline CHANGELOG.md:10]: choose branch on production/ staging

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+* Change icon in navigator when there is a validation error ([#87](https://github.com/Certora/vscode-certora-prover/pull/87))
```
Comment (sarittenen) [inline CHANGELOG.md:11]: + settings error icon
+error handling

File: src/settings/SolSettingsTab.svelte [hunk 3]
```diff
@@ -162,7 +164,10 @@
   }
 </script>
 
-<div class="card_parent_wrapper bg_dark border-rd">
+<div
+  class={'card_parent_wrapper bg_dark border-rd ' +
```
Comment (Biced) [inline src/settings/SolSettingsTab.svelte:168]: <div
  class='card_parent_wrapper bg_dark border-rd'
    class:disable_main="{$disableForm }"
>

**Review by Biced — CHANGES_REQUESTED** [review]

File: src/extension/utils/getRunInternalInfo.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+export async function getInternalDirPath() {
+  const path = workspace.workspaceFolders?.[0]
+  if (!path) return
+
+  const internalUri = Uri.parse(path.uri.path + CERTORA_INNER_DIR)
+  const checked = await checkDir(internalUri)
+  if (checked) {
+    const innerDirs = await workspace.fs.readDirectory(internalUri)
+    const dates = innerDirs.map(dir => {
+      if (dir[1] === 2) {
```
[the remark this answers is not in the document]
Comment (NoaBablikiCertora) [reply src/extension/utils/getRunInternalInfo.ts:33]: added explicit typing to "dir" in "getInternalDirPath"

**Pull request merged**